ai/groq/v0/config/definition.json

```diff
@@ -0,0 +1,18 @@
+{
+  "availableTasks": [
+    "TASK_TEXT_GENERATION_CHAT"
```

Comment:
I think for the sake for simplicity, if a model is supported across different vendors, we should only support it in one vendor? The whisper model is already supported through the OpenAI component.

Reply:
If I understand correctly, the main benefit that the users get from Groq is for speed rather than new models. right?

Reply:
Yeah, that's true. Their custom LPU chips provide much faster generation time. 
In that case, should we also support models like mixtral 22*8b?

Reply:
However, I don't think we could add audio-to-text task in time for this sprint. Since it require two interface (VDP-to-SDK and SDK-to-API) and tests to be implemented. I could add a new PR in the next sprint to add this task.

Reply:
Ah, because we are going to release in these days, we can support it in next Sprint! Thanks for researching!